dotnet/samples/KernelSyntaxExamples/Example42_KernelBuilder.cs

```diff
@@ -2,7 +2,7 @@
 
 // ==========================================================================================================
 // The easier way to instantiate the Semantic Kernel is to use KernelBuilder.
-// You can access the builder using either Kernel.Builder or KernelBuilder.
+// You can access the builder using either new KernelBuilder() or KernelBuilder.
```

Comment:
I think now there will be only one way how to access `KernelBuilder` - it's `new KernelBuilder()`. I think this comment can be removed.
```suggestion
```